pyproject.toml

```diff
@@ -0,0 +1,144 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "tianshou"
+version = "0.5.1"
+description = "A Library for Deep Reinforcement Learning"
+authors = ["TSAIL <trinkle23897@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/thu-ml/tianshou"
+classifiers = [
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+exclude = ["test/*", "examples/*", "docs/*"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+gymnasium = ">=0.29.0"
+h5py = ">=3.9.0"
+numba = ">=0.57.1"
+numpy = ">=1"
+packaging = "*"
+pettingzoo = ">=1.22"
+ray = ">=2"
+tensorboard = ">=2.5.0"
+# Torch 2.0.1 causes problems, see https://github.com/pytorch/pytorch/issues/100974
+torch = "==2.0.0"
```

Comment:
could you change to `>=2.0,!=2.0.1` if possible?

Reply:
Done in the other PR